plugins/source-kafka/README.md

```diff
@@ -0,0 +1,89 @@
+## kn-source_kafka
+
+`kn-source_kafka` is a plugin of Knative Client, which allows you to management of Kafka event source interactively from the command line.
```

Comment:
Why it is called `kn-source_kafka` and now `kn-source-kafka` ? I would expect a command like `kn source kafka` much like `kn source apiserver` or any other source.

Reply:
@maximilien mentioned that currently kn command, e.g. `kn source`, was not allowed to be extended. That is, I'm not allowed to add a subcommand `kafka` to `kn source`. But it can be changed.  We could talk about that in the meeting tomorrow.

Reply:
Yes, if this doesn't work for `kn` we should fix it. Its a key feature to make built-in source, plugin source and dynamical, generic source the same for the user experience (as it really doesn't matter for the user *how* a source is implemented, the usage across all sources should be uniform).

Reply:
Yup, we need to have a way to make exception for `kn source` group. Will open an issue and submit a PR for this tomorrow.

Reply:
See issue `client` issue [#814](https://github.com/knative/client/issues/814)